src/lsp/cobol_ptree/branching_statements.ml

```diff
@@ -99,6 +99,7 @@ and perform_inline_stmt =
   }
 
 and perform_mode =
+  | PerformForever
```

Comment:
Please add this as last constructor as it's not so usual, with a `(* COBOL2002, according to GnuCOBOL's sources *)` comment (this is not even mentioned in COBOL2014, so I'm a bit surprised).

Reply:
Now that I think about it, the comment was attached to the `FOREVER` keyword itself, that appears somewhere else in the grammar.
In this particular case this seems a Micro-focus extension.

Reply:
Not even Micro-focus, just COBOL-IT…

Reply:
So the comment should just state: `GC extension` (GC standing for GnuCOBOL).

Reply:
Do you mean the comment in GnuCOBOL's sources is wrong? How do you know it is just a COBOL-IT extension?

Reply:
I've looked for it in several documents. The only place I could find it is on a reference for COBOL-IT.

My (almost sure) guess is: the `COBOL2002 C/S` comment comes from the introduction of the keyword `FOREVER` in the `RETRY` context. It was later added to the `PERFORM` context without the comment being adjusted.